tests/estimator_test.py

```diff
@@ -37,34 +38,34 @@ def test_estimates() -> None:
     c.Z(2)
     pbox = PauliExpBox([Pauli.X, Pauli.I, Pauli.Z], 0.25)
     c.add_pauliexpbox(pbox, [2, 0, 1])
-    c = b.get_compiled_circuit(c, 0)
-    resources = b.get_resources(c)
-    assert resources["CNOT"] >= 1
-    assert resources["QubitClifford"] >= 1
-    assert resources["R"] >= 1
-    assert resources["T"] >= 1
-    assert resources["Depth"] >= 1
-    assert resources["Width"] == 3
-    assert resources["BorrowedWidth"] == 0
+    # c = b.get_compiled_circuit(c, 0)
```

Comment:
Maybe add a code comment referencing the new issue so that it's clear why these tests are disabled (same below).

Reply:
done, I have added one to the class as well.